implementation/src/main/java/io/smallrye/mutiny/groups/UniCreate.java

```diff
@@ -186,8 +186,8 @@ private UniCreate() {
      * {@link Supplier}. Unlike {@link #deferred(Supplier)}, the supplier produces an item and not an {@link Uni}.
      * <p>
      * If the supplier produces {@code null}, {@code null} is used as item event.
-     * If the supplier throws an exception, a failure event with the exception is fired. If the supplier produces
-     * {@code null}, a failure event containing a {@link NullPointerException} is fired.
+     * If the supplier throws an exception, a failure event with the exception is fired.
+     * If the supplier is {@code null}, a failure event containing a {@link NullPointerException} is fired.
```

Comment:
I don't believe it's right.
If the supplier is `null`, an illegal argument exception is thrown synchronously.

The first line of the method is: Supplier<? extends T> actual = ParameterValidation.nonNull(supplier, "supplier");

Reply:
Oh right, so here's a fix for the fix